sql/updates/world/2016_01_25_09_world.sql

```diff
@@ -0,0 +1,56 @@
+(4488, 0,  2,  0, 52, 0, 100, 0,    0, 4488, 0, 0,  1,    4, 3000, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Text 0 Over - Say Text 4"),
+(4488, 0,  3,  0, 52, 0, 100, 0,    4, 4488, 0, 0, 11, 6767,    0, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Text 4 Over - Cast spell 6767 'Mark of Shame'"),
+(4488, 0,  4,  5, 62, 0, 100, 0, 4764,    1, 0, 0, 72,    0,    0, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Gossip Option 1 selected - Close Gossip"),
+(4488, 0,  5,  0, 61, 0, 100, 0,    0,    0, 0, 0,  1,    1, 5000, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Gossip Option 1 selected - Say Text 1"),
+(4488, 0,  6,  0, 52, 0, 100, 0,    1, 4488, 0, 0,  1,    4, 3000, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Text 1 Over - Say Text 4"),
+(4488, 0,  7,  0, 52, 0, 100, 0,    4, 4488, 0, 0, 11, 6767,    0, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Text 4 Over - Cast spell 6767 'Mark of Shame'"),
+(4488, 0,  8,  9, 62, 0, 100, 0, 4764,    2, 0, 0, 72,    0,    0, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Gossip Option 2 selected - Close Gossip"),
+(4488, 0,  9,  0, 61, 0, 100, 0,    0,    0, 0, 0,  1,    2, 5000, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Gossip Option 2 selected - Say Text 2"),
+(4488, 0, 10,  0, 52, 0, 100, 0,    2, 4488, 0, 0,  1,    4, 3000, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Text 2 Over - Say Text 4"),
+(4488, 0, 11,  0, 52, 0, 100, 0,    4, 4488, 0, 0, 11, 6767,    0, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Text 4 Over - Cast spell 6767 'Mark of Shame'"),
+(4488, 0, 12, 13, 62, 0, 100, 0, 4764,    3, 0, 0, 72,    0,    0, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Gossip Option 3 selected - Close Gossip"),
+(4488, 0, 13, 14, 61, 0, 100, 0,    0,    0, 0, 0,  1,    3, 5000, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Gossip Option 3 selected - Say Text 3"),
+(4488, 0, 14, 15, 61, 0, 100, 0,    0,    0, 0, 0, 15, 6628,    0, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Gossip Option 3 selected - Credit Quest 6628 'Test of Lore'"),
+(4488, 0, 15,  0, 61, 0, 100, 0,    0,    0, 0, 0,  5,    2,    0, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Gossip Option 3 selected - Play Emote 2 (ONESHOT_BOW)");
+
+-- conditions showing gossip menu and gossip menu options only when Quest 6628 is active
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId`= 14 AND `SourceGroup`= 4764 AND `SourceEntry`= 5822;
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId`= 15 AND `SourceGroup`= 4764 AND `SourceEntry` IN (0,1,2,3,4);
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`,`SourceGroup`,`SourceEntry`,`SourceId`,`ElseGroup`,`ConditionTypeOrReference`,`ConditionTarget`,`ConditionValue1`,`ConditionValue2`,`ConditionValue3`,`ErrorTextId`,`ScriptName`,`Comment`) VALUES
+(14, 4764, 5822, 0, 0, 9, 0, 6628, 0, 0, 0, '', 'Show gossip dialog text 5822 if Quest 6628 is Active (taken)'),
```

Comment:
OK, thanks for info. I will include that condition. I just found it odd that it seems to work without a condition, as if the quest_template_addon has determined the gossip flow for the quest text.